region/maridia/outer/Fish Tank.json

```diff
@@ -714,7 +714,20 @@
         "canSuitlessMaridia",
         "HiJump",
         {"or": [
-          "canCarefulJump",
+          "canTrickyJump",
+          "canDodgeWhileShooting",
```

Comment:
This is the bottom pirate at the end of the door's runway.  There are a couple easy ways to avoid it, but not intuitively.  Shooting at it, scrolling the camera while it walks left, cutting momentum and morphing after falling next to the pirate, or jumping over.  And jumping over isn't reliable.